### PR TITLE
Password reset APIs

### PR DESCRIPTION
**Pull request by sandygudie**
# Description
Added Password reset API. The forget password endpoint sends a reset link to the user's email and generates a password reset token. The reset password endpoint uses token (req.query)  and new password( req.body) to reset user password . New password generated is encrypted and stored in database.

## Type of change
- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update
- [x] This change requires touching the App.js file

# Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

Ticket NO 140

Link to endpoints testing on postman
[forget password](https://res.cloudinary.com/dvpoiwd0t/image/upload/v1593335257/forget_password_done_fnsgsn.jpg)
[resetpassword](https://res.cloudinary.com/dvpoiwd0t/image/upload/v1593335258/resetsuccessful_czhrdr.jpg)

### CODE REVIEW

**Comment by chinazaaa** [comment]
Send screenshots of the result

**Comment by sandygudie** [comment]
There are links of Postman test screenshots on the PR

File: controllers/phone_call_api.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ app.use(bodyParser.urlencoded({ extended: false }));
 
 const accountSid = process.env.TWILIO_SID;
 const auth = process.env.TWILIO_AUTH;
-const twilioClient = require("twilio")(accountSid, auth);
+// const twilioClient = require("twilio")(accountSid, auth);
```
Comment (nerdyphil) [inline controllers/phone_call_api.js:13]: please uncomment this

**Comment by sandygudie** [comment]
Done

**Pull request closed without merging**